src/webpage/embed.ts

```diff
@@ -157,13 +157,11 @@ Url.pathname.split("/")[Url.pathname.split("/").length - 1];
 			if(this.json?.footer?.text){
 				const span = document.createElement("span");
 				span.textContent = this.json.footer.text;
-				span.classList.add("spaceright");
```

Comment:
Why did this get removed?

Reply:
The class doesn't have any css, you can put it back

Reply:
I wasn't asking for it back, but that's odd lol, I trust you on this one